sql/core/src/test/scala/org/apache/spark/sql/JoinHintSuite.scala

```diff
@@ -612,8 +612,8 @@ class JoinHintSuite extends PlanTest with SharedSparkSession with AdaptiveSparkP
 
         val logs = hintAppender.loggingEvents.map(_.getRenderedMessage)
           .filter(_.contains("is not supported in the query:"))
-        assert(logs.size == 2)
-        logs.forall(_.contains(s"build left for ${joinType.split("_").mkString(" ")} join."))
+        assert(logs.size == 2 &&
+          logs.forall(_.contains(s"build left for ${joinType.split("_").mkString(" ")} join.")))
```

Comment:
how about 
```
assert(logs.size == 2)
assert(logs.forall(_.contains(s"build left for ${joinType.split("_").mkString(" ")} join.")))
```

It might be easier to distinguish which statement is false if something goes wrong.

Reply:
Actually maybe even
```
logs.forall(log => assert(log.contains...))
```
Also: `assert(logs.size === 2)`

Reply:
@srowen, @advancedxy - updated, thanks.